docs/Migrating-v4.md

```diff
@@ -1,17 +1,19 @@
 # ECL v4 migration notes
 
-The following guidelines aim to facilitate migration between ECL v3 to v4.
+The following guidelines aim to facilitate migration between ECL v3 and v4.
 
 - [Js modifications](#js-modifications)
 - [Style modifications](#style-modifications)
 - [Component modifications](#component-modifications)
+- [Customizing the display](#customizing-the-display)
 
 ## Js modifications
 
 - The ECL javascript is not bundling anymore `pikaday`, therefore this library needs to be loaded by the project using ECL, it is only
   needed when a `datepicker` instance is present in the page.
-- There is no hard dependency on `moment.js` anymore, this library is only needed when customising the default format for the dates, the default
-  in ECL is `DD-MM-YYYY`. If the format is customised but moment is not loaded there will not be errors in the console but the chosen format would not be consistently shown.
+- There is no hard dependency on `moment.js` anymore, this library is only needed when customising the default format for the dates, the default in ECL is `DD-MM-YYYY`. If the format is customised but moment is not loaded there will not be errors in the console but the chosen format would not be consistently shown.
+- Callback are now provided for some components, and can be used to trigger custom code when needed
```

Comment:
```suggestion
- Callback are now provided for some components, and can be used to trigger custom code when needed
```
A simple event manager is now available in ECL, some of the components are defining and emitting events that the user can subscribe to and execute a callback when they are emitted. The list of these events and component supporting them will increase in the future.